### PR TITLE
Add ImportTiffPipeline

### PR DESCRIPTION
**Pull request by zigaLuksic**
Adds a pipeline for importing tiffs to EOPatches
Also includes tests

### CODE REVIEW

File: tests/test_pipelines/test_import_tiff.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    [
+        "import_tiff_temporal",
+        "import_tiff_timeless",
+        pytest.param("import_tiff_resized_new_size", marks=pytest.mark.chain),
+        "import_tiff_resized_scale_factors",
+    ],
+)
+def test_import_tiff_pipeline(folders, experiment_name):
+    # Can't use utility testing due to custom pipeline
+    config_filename = os.path.join(folders["config_folder"], experiment_name + ".json")
```
Comment (veseln) [inline tests/test_pipelines/test_import_tiff.py:33]: My preference is f-strings when concatenating strings. But can also live with this

File: eogrow/pipelines/import_tiff.py [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+from ..core.pipeline import Pipeline
+from ..core.schemas import BaseSchema
+from ..utils.filter import get_patches_with_missing_features
+from ..utils.types import Feature
+from ..utils.validators import optional_field_validator, parse_dtype
+
+
+class ResizeSchema(BaseSchema):
+    """Uses resolution instead of new_size or scale_factors."""
```
Comment (veseln) [inline eogrow/pipelines/import_tiff.py:21]: Not sure what this description tries to convey, maybe try to make it clearer.

**Review by veseln — COMMENTED** [review]
Minor comments. Otherwise looks okay, ran the tests locally and they seem to work.

File: eogrow/pipelines/import_tiff.py (hunk 2 above)
Comment (zigaLuksic) [reply eogrow/pipelines/import_tiff.py]: Old text, fixed

File: eogrow/pipelines/import_tiff.py [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+        kind, params = v
+        if kind is ResizeParam.NEW_SIZE:
+            params = (round(params[0]), round(params[1]))
+        return kind, params
+
+
+class ImportTiffPipeline(Pipeline):
+    class Schema(Pipeline.Schema):
+        output_folder_key: str
+        tiff_folder_key: str = "input_data"
```
[the remark this answers is not in the document]
Comment (zigaLuksic) [reply eogrow/pipelines/import_tiff.py]: Good catch, removed and updated tests

File: eogrow/pipelines/import_tiff.py [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+        input_filename: str
```
[the remark this answers is not in the document]
Comment (zigaLuksic) [reply eogrow/pipelines/import_tiff.py]: fiiiiine :P added

File: tests/test_pipelines/test_import_tiff.py (hunk 1 above)
Comment (zigaLuksic) [reply tests/test_pipelines/test_import_tiff.py]: changed to f-strings
they are a bit nicer indeed

**Pull request merged**